Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -277,6 +277,45 @@
     <LocalizationCode>furn_metal_smelter</LocalizationCode>
     <UnlocalizedDescription>furn_metal_smelter_desc</UnlocalizedDescription>
   </Furniture>
+	
+<Furniture objectType="Metal Furnace">
+    <Name>Metal Furnace</Name>
+    <Description>Smelts your raw ores into metal.</Description>
+    <TypeTag>Processor</TypeTag>
+    <MovementCost>1</MovementCost>
+    <PathfindingModifier>1.5</PathfindingModifier>
+    <PathfindingWeight>2</PathfindingWeight>
+    <Width>3</Width>
+    <Height>3</Height>
+
+    <BuildingJob jobTime="0" />
+
+    <JobSpotOffset X="1" Y="0" />
+
+	<FactoryInfo>
+		<PossibleProductions>
+			<ProductionChain name="Iron smelting" processingTime="4">
+				<Input>
+					<Item objectType="Raw Iron" amount="3" slotPosX="0" slotPosY="0" />
```

Comment:
Factorial holds the material that has been fed to the machine inside it and you have to click on it to see how much it's holding. Same thing with material dropped of at a building site in Rimworld. I think that is a better solution. 

The reason I asked about 3 inputs is because we'd need 4 tiles to put and get material from plus the sprite of the machine, making them very big. 

It's a personal preference and something that I think is worth discussing separately.